src/utils/time-entry-utils/timeEntryUtils.js

```diff
@@ -0,0 +1,15 @@
+import dayjs from 'dayjs';
+
+export const getSingleTimeEntryResponseItem = (timeEntryResponseData) => {
+  if (timeEntryResponseData.items && timeEntryResponseData.items.length > 0)
+    return timeEntryResponseData.items[0];
+  else return null;
```

Comment:
Should we throw an error here as well to inform a developer what the issue is?

Reply:
Ah yeah I was contemplating this. Think you're right.
I think on the 'else' we should not return anything and just throw an error as then it conforms to the name/purpose of this function.

Reply:
done